cmd/repl/repl.go

```diff
@@ -81,15 +81,13 @@ func render(rsp interface{}, table *tablewriter.Table) bool {
 		table.Append(row)
 		isTable = true
 	case error:
-		if os.Getenv("SQLFLOW_log_dir") != "" { // To avoid printing duplicated error message to console
-			log.New(os.Stderr, "", 0).Printf("ERROR: %v\n", s)
```

Comment:
We'd better keep this check somewhere to avoid printing error message twice when logging to stderr.

Reply:
Maybe we can use `log.Fatalf` to exit if some errors.

Reply:
`log.Fatalf` will terminate the program, that's no good for the interactive mode of REPL.